{"additions": 4, "auther_ref": "CMSSW_13_2_X", "auther_sha": "a4173651c33b0258447acbb332cfbf6ee5c2e539", "author": "alikaanguven", "body": "PR description:\r\nAddressing the issue:\r\nhttps://github.com/cms-sw/cmssw/issues/47777\r\n\r\nPR validation:\r\nStill python3. It should work.\r\n\r\nOrigin of backport:\r\nhttps://github.com/cms-sw/cmssw/pull/44201", "branch": "CMSSW_13_2_X", "changed_files": 1, "comments": 7, "commits": 1, "created_at": "1743694005", "deletions": 4, "labels": ["core-approved", "fully-signed", "tests-approved", "orp-pending", "backport-ok"], "milestone": "CMSSW_13_2_X", "number": 47786, "release-notes": [], "review_comments": 0, "state": "open", "title": "Backport LumiList.py bugfix to CMSSW_13_2_X", "updated_at": "1743704856", "user": "alikaanguven"}